_data/sites.yml

```diff
@@ -1577,6 +1577,11 @@
   size: 163
   last_checked: 2022-04-26
 
+  - domain: joshualiu.org
```

Comment:
It appears there's an extra indent here. Do you mind fixing that?🙂

Reply:
Misread "indent" as "incident" on your comment.
Apologies,  I'm a hobbyist that doesn't use github or the tools here that often.

Reply:
All good! Thanks for adding your site🙂